src/pages/ForgotPassword/ForgotPassword.tsx

```diff
@@ -52,13 +52,11 @@ const ForgotPassword = () => {
   return (
     <AuthContainer>
       <AuthCard>
-        <AuthForgetHead>
-          Yo! Forgot your <br /> password again?!
-        </AuthForgetHead>
+        <AuthHead>Forgot Password</AuthHead>
         <KeyIcon />
-        <AuthSub>
-          No worries! <br /> Enter your email and we will send you a reset
-        </AuthSub>
+        <AuthSpan>
+          Enter your email and we will send you a reset password link
```

Comment:
Could we make this sound a little more casual? Maybe "Enter your email to get a password reset link."